pikelet/src/surface/grammar.lalrpop

```diff
@@ -39,11 +42,26 @@ pub Term: Term<&'input str> = {
     <term: ExprTerm> ":" <ty: Term> => Term::Ann(Box::new(term), Box::new(ty)),
 };
 
+Sequence<Item>: Vec<Item> = {
+    <mut bindings: (<Item> ";")+> <last: Item?> => {
+        bindings.extend(last);
+        bindings
+    }
+}
+
+Item: (Range<usize>, &'input str, Term<&'input str>) = {
+  TermEntry,
+  TypeEntry,
+}
```

Comment:
I'm guessing we could have an `Item` enum here, that can either be a `Declaration` or a `Definition`?

Reply:
:+1: will look into it tomorrow, if I can

Reply:
So, I was unsure about whether `RecordType`/`RecordTerm` should be returning this `Item`,
The conclusion I came to was that the answer is no, currently `RecordType` would _always_ yield an `Item::Declaration`, and `RecordTerm` an `Item::Definition`.

And wrapping these cases into an enum just introduces a match leg, where there doesn't need to be one.  Does that sound right?